TeamCode/src/main/java/org/firstinspires/ftc/teamcode/api/linear/LinearOutTake.kt

```diff
@@ -0,0 +1,11 @@
+package org.firstinspires.ftc.teamcode.api.linear
+
+import org.firstinspires.ftc.teamcode.api.OutTake
+
+object LinearOutTake : LinearAPI() {
+    fun beltPosition(final: Int) {
+        while (OutTake.outTakeBelt.position < final) {
```

Comment:
Could you add `&& opModeIsActive()` so that you can cancel a `beltPosition` call with the stop button?

Reply:
done